backend/src/Services/Logging/TestLoggerImpl.ts

```diff
@@ -0,0 +1,70 @@
+import { randomUUID } from "crypto";
+import { ILoggingContext } from "./ILogger";
+import Logger from "./Logger";
+
+
+export class TestLoggerImpl {
+
+    logs:string[] = [];
+
+    constructor() {
+    }
+
+    debug(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
+        this.log(context, "", message, ...optionalParams);
```

Comment:
Should debug and info also have the "DEBUG" and "INFO" prefixes?

Reply:
I just copied what's in the normal logger.

My general thought was that the debug and info logs would be the "normal" ones.  We would want warn and errors to stand out more.

However, if we're using new relic to filter things anyway, may make sense to add the prefix?